### ISSUE TITLE
Double newlines get deleted after sending the answer

### ISSUE THREAD

**Issue by mangoepic** [LAION-AI/Open-Assistant#1690]
Double new lines like

this one above, would be deleted in an answer. I'm not sure if it's a bug but I've wanted to use it to separate paragraphs / make maths more readable.

**Comment by bitplane** [comment]
The answers are in markdown, just like on here. So I guess you'll need to do:

```markdown
Something like this
<br>
for it to work
```
i.e.:

Something like this
<br>
for it to work

Though inserting HTML into answers might cause other issues, there's a very limited subset that are safe, and it'll probably come out encoded instead.

**Comment by bitplane** [comment]
Looks like it's a rendering issue anyway. Firstly, double newlines are saved. Secondly, `<br>` doesn't get mangled, but it doesn't get rendered either:

![image](https://user-images.githubusercontent.com/273059/219924355-3803a07e-5e57-4e39-9dce-30ac01840330.png)

![image](https://user-images.githubusercontent.com/273059/219924662-edaabd1a-4b0c-4a69-b4ec-16a95b6c8c80.png)

```json
{"text":"This is the prompt\n\n<br>\n\nThis comes after it"}
```
I'm thinking that if we're doing markdown, then we should probably support inline `<br>` tags and allow that.

**Comment by olliestanley** [comment]
This is just a matter of actually showing the newlines in the UI I think. They are definitely present in the DB

**Comment by GuilleHoardings** [comment]
I think this was fixed in #1746.
